gamla/debug.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+import builtins
+import logging
+from typing import Text
+
+from toolz import curried
+
+logger = curried.do(logging.info)
+
+
+def log_text(text: Text, level: int = logging.INFO):
+    return curried.do(lambda x: logging.log(level, text.format(x)))
```

Comment:
Same.
Also, why do we need both `log_text` and `logger`?

Reply:
hmm I can guess that `logger` lets you just use it without giving text